docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -36,25 +34,52 @@ forc deploy <signing-key> --node-url 127.0.0.1:4000/graphql
 Or to deploy without using a signing key:
 
 ```sh
-forc deploy --unsigned --node-url 127.0.0.1:4000/graphql
+forc deploy --default-signer --node-url 127.0.0.1:4000/graphql
```

Comment:
it seems like this doesn't work ootb anymore, and the wallet has to be funded? probably worth noting here